generate-markdown-readme

```diff
@@ -60,15 +60,39 @@ try {
 		}
 	}
 	if ( $github_account_repo ) {
+		$env_file = $readme_root . '/.ci-env.sh';
+		if ( file_exists( $env_file ) ) {
+			$lines = file( $env_file, FILE_IGNORE_NEW_LINES );
```

Comment:
@valendesigns this is a bit of a hack, but we can actually leverage `parse_ini_file()` to parse the `.ci-env.sh` file.